daemon/algod/api/server/v2/account.go

```diff
@@ -125,8 +125,13 @@ func AccountDataToAccount(
 		AuthAddr:                    addrOrNil(record.AuthAddr),
 		AppsLocalState:              &appsLocalState,
 		AppsTotalSchema:             &totalAppSchema,
-		AppsTotalExtraPages:         &totalExtraPages,
```

Comment:
you could have just called
```golang
AppsTotalExtraPages:         numOrNil(totalExtraPages),
```

Reply:
thanks, did not know about this helper, fixed!